content/articles/index.md

```diff
@@ -0,0 +1,71 @@
+  - [Backward Ray Tracing Algorithm](#backward-ray-tracing) 
+ - [Conclusion](#conclusion)
+
+
+### What is Ray Tracing Algorithm and how does it work?
+Ray Tracing Algorithm is a very effective way of drawing synthesizing images. It's a method of rendering, this process has the potential to produce a much greater degree of authenticity that is probably higher than that of other scanners. However, this involves a huge calculation cost. Computer imagery has become a very basic need in today's digital world. Computer graphics come into play when we need to create an image that looks real and indistinguishable from the standard image taken. There are many ways to accomplish this. Ray Tracing is one such way to give. This is used to produce an image by tracing the light path such as pixels in an image plane and mimicking its effects when encountering objects.
+
+
+The process aims to bring real light to the simulation by imitating the behavior of light intensity. It is done by reflecting the light rays from the camera, ‘following’ them until they encounter an object, and counting where each ray of light cuts an object in the square. This then allows it to calculate how the pixel is affected by the light rays, and to keep track of the ray as it jumps on the scene. This requires the computer to track a certain number of rays from each intersection to the reflected light model, thus creating a growing number of rays in the square.
+
+
+
+### History of Ray Tracing
+It was created by Robert Goldstein and Arthur Appel in the 1960s. According to [Wikipedia](https://en.wikipedia.org/wiki/Ray_tracing_(graphics)#Algorithm_overview), in 1968, Arthur Appel was the first to use a computer to generate shaded images using ray tracing. He used ray tracing to figure out which surface was closest to the camera at each image point, then traced secondary rays to the source of light from each shaded point to see if it was in shadow or not.
+
+Later, by simulating the photographic process in reverse, Robert Goldstein used ray tracing is to create shaded images of solids. He used a non-recursive ray tracing based algorithm known as ray casting. Ray tracing is widely regarded as the most effective method of image synthesis.
+
+
+### Types of Ray Tracing Algorithm
+Various methods are induced to make ray tracing effective. There are three types of ray tracing algorithms, they include; hybrid ray tracing algorithm, forward ray tracing algorithm, backward ray tracing algorithm.
+
+
+#### 1. Hybrid Ray Tracing Algorithm
+It's a combination of backward and forward ray tracing techniques hence making it more effective since it has the benefits of both techniques.   
+
+A first pass through the scene is rendered using a basic raster image processing system such as [Vulkan](https://vulkan-tutorial.com/). The first pass is then followed by a ray tracing phase which adds more information to the image created in the first pass.
+

```

Comment:
remove the extra spaces. There should be only one space between the paragraphs or sections.